GlCanvas.js

```diff
@@ -0,0 +1,187 @@
+(function() {
+function GlPlotter() {
+    this.pluginName = 'glplotter';
+    this.pluginVersion = '0.2';
+
+    /**
+     * Initialize a webgl layer
+     */
+    this.init = function(plot) {
+        var defaultPlotOffset = {
+            left: 0,
+            right: 0,
+            top: 0,
+            bottom: 0
+        };
+
+        var textures = [],
+            materials = [],
+            geometries = [];
+
+        plot.hooks.processOptions.push(processOptions);
+
+        function processOptions(plot, options) {
+            var container = plot.getPlaceholder(),
+                webglsurface = plot.getWebGlSurface(),
+                canvas = plot.getWebGlCanvas(),
+                width = webglsurface.width,
+                height = webglsurface.height,
+                renderer = new THREE.WebGLRenderer({ canvas: canvas, antialias: false, alpha: true }),
+                mainScene = new THREE.Scene(),
+                camera = new THREE.OrthographicCamera(webglsurface.width / 2, -webglsurface.width / 2, -webglsurface.height / 2, webglsurface.height / 2, 0.1, 1000),
+                cameraFocus = new THREE.Vector3(width / 2, height / 2, 1000);
```

Comment:
Could you please add a local variable for the 1000? I guess this is the z index, but not sure.